contracts/locker/BaseLocker.sol

```diff
@@ -50,6 +52,11 @@ contract BaseLocker is
     IERC20 public underlying;
     IOmnichainStaking public staking;
 
+    event TokenAddressSet(address indexed oldToken, address indexed newToken);
```

Comment:
Would be great if we can move these events to the interface.

Reply:
Done https://github.com/zerolend/governance/pull/35/commits/bb666f7d793810a5dcc7b0c374111c15272b7dde